### PR TITLE
Reference ../utils/util instead of /Util (which doesn't seem to exist)

### PR DESCRIPTION
**Pull request by pbjorklund**
Just pulled the repo and saw that ```gulp test``` didn't work. That made me sad.

Seems the implementation files were referencing an invalid location.

I like the idea behind this repo. I'l be back after I go learn typescript.

### CODE REVIEW

**Comment by pbjorklund** [comment]
Fixed a failing test where CT getById returned an unquoted guid.

(Should probably be a separate PR but was to much wrangling for me to create it against the test-broken-dev-branch)

File: src/sharepoint/provisioning/objecthandlers/objectcomposedlook/objectcomposedlook.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 "use strict";
 
 /// <reference path="..\schema\icomposedlook.d.ts" />
-import { replaceUrlTokens, getRelativeUrl } from "../../../Util";
+/// <reference path="../../../utils/util" />
```
Comment (pbjorklund) [inline src/sharepoint/provisioning/objecthandlers/objectcomposedlook/objectcomposedlook.ts]: Obviously this is redudant. Waiting to see what the group decides the "right" way to do it is.

File: src/sharepoint/provisioning/objecthandlers/objectlists/objectlists.ts [hunk 2]
```diff
@@ -257,7 +257,7 @@ export class ObjectLists extends ObjectHandlerBase {
         });
     }
     private GetFieldXmlAttr(fieldXml: string, attr: string) {
-        return $(jQuery.parseXML(fieldXml)).find("Field").attr(attr);
+        return jQuery(jQuery.parseXML(fieldXml)).find("Field").attr(attr);
```
Comment (pbjorklund) [inline src/sharepoint/provisioning/objecthandlers/objectlists/objectlists.ts]: This is just for consistency. From what I can see we reference ```jQuery``` everywhere else. I still belive we should reference neither $ or jQuery :)

File: src/sharepoint/rest/web/lists/contenttypes/contenttypes.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ export class ContentTypes extends Queryable {
         super(url, "/ContentTypes");
     }
     public getById(id: string) {
-        this._url.push(`(${id})`);
+        this._url.push(`(\"${id}\")`);
```
Comment (pbjorklund) [inline src/sharepoint/rest/web/lists/contenttypes/contenttypes.ts]: Test required that we got "GUID", not just GUID which this returned before.
The actual way I did the string interpolation here to get "GUID" seems a bit wonky tho. Better ideas?

File: tsconfig.json [hunk 4]
```diff
@@ -1,9 +1,12 @@
...
-        "module": "umd",
-        "jsx": "react",
+         "module": "umd",
+         "jsx": "react",
         "declaration": true,
         "sourceMap": true
-    }
+    },
+    "files": [
+        "typings/main.d.ts"
```
Comment (pbjorklund) [inline tsconfig.json]: According to the typings docs, if you reference like this you shouldn't need to use ```/// <reference``` in TS files. I'm unclear on wether or not this works tho. But that probably has more to do with my massive 2 days of typescript experience than anything else.

**Comment by patrick-rodgers** [comment]
Thanks for this, looks good.

**Pull request merged**